tests/safeds/data/tabular/containers/_table/_tagged_table/test_filter_rows.py

```diff
@@ -0,0 +1,27 @@
+from safeds.data.tabular.containers import TaggedTable
+
+
+def test_should_remove_row() -> None:
+    table = TaggedTable(
+        {
+            "feature_1": [3, 9, 6],
+            "feature_2": [6, 12, 9],
+            "target": [1, 3, 2],
+        },
+        "target",
+        None
+    )
+    new_table = table.filter_rows(lambda row: all(row.get_value(col) < 10 for col in table.column_names))
+    expected = TaggedTable(
+        {
+            "feature_1": [3, 6],
+            "feature_2": [6, 9],
+            "target": [1, 2],
+        },
+        "target",
+        None
+    )
+    assert new_table.schema == expected.schema
+    assert new_table.features == expected.features
+    assert new_table.target == expected.target
+    assert new_table == expected
```

Comment:
Might be worth implementing `__eq__` for `TaggedTable` first so only one assertion is needed.

Reply:
I thought we had deliberately decided against that, so failing tests would produce more helpful output.

Wasn't this the point of #124?

Reply:
Works for me, too.

Reply:
Thinking more about this, it could make sense to extract it into its own helper method in `tests/helpers`, though.

Reply:
Did extract it into its own helper method.